### ISSUE TITLE
zlib:x86-windows-static build failure

### ISSUE THREAD

**Issue by TyOverby** [microsoft/vcpkg#12573]
**Host Environment**
- OS: Windows
- Compiler: unknown

**To Reproduce**
```bash
git clone --depth=1 https://github.com/Microsoft/vcpkg.git
vcpkg\bootstrap-vcpkg.bat
vcpkg\vcpkg install libpng:x86-windows-static
```

**Failure logs** 
[Full Failure Log](https://pipelines.actions.githubusercontent.com/csCJ9hOMQLuRokZJ3YeMTTb0Vo7X3zPENCSV9mTCdxBUTimfv6/_apis/pipelines/1/runs/16/signedlogcontent/3?urlExpires=2020-07-25T21%3A25%3A18.1385954Z&urlSigningMethod=HMACV1&urlSignature=LcnoQg59B6AzqC0wqpmmud6fYXC%2F7atnrP4vx2Mkfc4%3D)

Important snippet: 

```
2020-07-25T21:20:22.2453754Z CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:279 (find_program):
2020-07-25T21:20:22.2454006Z   Could not find PKGCONFIG using the following names: pkg-config
2020-07-25T21:20:22.2454233Z Call Stack (most recent call first):
2020-07-25T21:20:22.2454442Z   ports/zlib/portfile.cmake:46 (vcpkg_fixup_pkgconfig)
2020-07-25T21:20:22.2454645Z   scripts/ports.cmake:79 (include)
2020-07-25T21:20:22.2454942Z 
2020-07-25T21:20:22.2455147Z Error: Building package zlib:x86-windows-static failed with: BUILD_FAILED
2020-07-25T21:20:22.2455368Z Please ensure you're using the latest portfiles with `.\vcpkg update`, then
2020-07-25T21:20:22.2455633Z submit an issue at https://github.com/Microsoft/vcpkg/issues including:
2020-07-25T21:20:22.2455857Z   Package: zlib:x86-windows-static
2020-07-25T21:20:22.2456055Z   Vcpkg version: 2020.06.15-nohash
```

**Comment by StarGate-One** [comment]
See issue #12565

**Comment by ngovankhoa** [comment]
The same for me. The old version 2020.04 works well.

**Comment by TyOverby** [comment]
@ngovankhoa how do you install a specific version of the library?

**Comment by ngovankhoa** [comment]
> 
> 
> @ngovankhoa how do you install a specific version of the library?

After cloning vcpkg, you need to go go vcpkg folder and run checkout command:
`git checkout 2020.04`

**Comment by TyOverby** [comment]
Closed in favor of #12565
